requirements.txt

```diff
@@ -32,3 +32,6 @@ sentry-sdk # logging
 thop  # FLOPs computation
 
 sparseml==1.2.0
+
+websockets==10.4
+openvino-dev[onnx]
```

Comment:
Might be good to specify 2022.1 or 2022.3 version here just in case.

Reply:
But if I remember correctly, the blobconverter doesn't support OpenVINO 2022.3, the latest version OpenVINO that is supports is 2022.1

Reply:
Yeah, it's not yet updated. Let's set it to 2022.1 :)

Reply:
Updated